### ISSUE TITLE
Timer seems to be 10 minutes off.

### ISSUE THREAD

**Issue by mr-jak3z** [Antony256/OceanTrip#3]
The timer to register for ocean fishing seems to be 10 minutes delayed, it always registers at XX:10 which results in most groups being "small" consisting of 4-8 people at most, while if you could queue at XX:00 the groups could be of 24.
Any way to adjust it?

**Comment by Antony256** [comment]
The game scales the spectral pop rate according to the group size. I have been getting better results with smaller groups, possibly because there are less people to screw things up with wrong baits.

**Comment by mr-jak3z** [comment]
I see, fair enough! I’m just struggling getting past the 10-12k points. And
I seem to not be using double hook too often while on spectral currents.

I will keep trying with the 10 minutes offset! I love your work with this
one !

**Comment by mr-jak3z** [comment]
Do you have any idea what it would be that sometimes the botbase doesn't use DoubleHook or any GP at all during spectral currents? And some other times it uses skills and Hi Cordials.

**Comment by Antony256** [comment]
I'm checking for weather conditions and bite times before using skills. I'll double-check I'm not missing any fish.

**Comment by mr-jak3z** [comment]
Thanks!
It might just be my bad luck, but in the last ... 2 days I haven't gotten any doublehook that I could see or any other GP usage from the bot more than 1 Mooch or 2.

**Comment by mr-jak3z** [comment]
Wanted to give you a bit more of feedback, not sure if I can provide you logs or they would be of any interest for you.
I have parsed the logs of 14 days of using this botbase, it used 13 times Double Hook in 13 logs, and only in one log it used it  15 times. The day it got used 15 times was an extraordinary occasion as I had queued myself to do it manually but I got called into something and I enabled to bot hoping it could work even if I started manually and it did work, and it was my best result to date perhaps coincidence. 
I also don't seem to see it ever use Surface Slap or Identical Cast. Maybe it's too strict in the conditions to use that as well?
